pombola/kenya/static/js/facebook-experiment.js

```diff
@@ -0,0 +1,170 @@
+// http://stackoverflow.com/questions/901115/how-can-i-get-query-string-values-in-javascript
+function getParameterByName(name, url) {
+    if (!url) url = window.location.href;
+    name = name.replace(/[\[\]]/g, "\\$&");
+    var regex = new RegExp("[?&]" + name + "(=([^&#]*)|&|#|$)"),
+        results = regex.exec(url);
+    if (!results) return null;
+    if (!results[2]) return '';
+    return decodeURIComponent(results[2].replace(/\+/g, " "));
+}
+
+$(function(){
+
+    // Show the hidden page element (we don't show anything if no JS is available)
+    $('#ctas').show();
+
+    var options = {
+            placeholder: 'Constituency',
+            allowClear: true,
+        },
+
+        modelMatcher = function (term, text) {
```

Comment:
The code in `modelMatcher` is also a duplicate of the definition in `pombola/kenya/templates/iebc_office_single_select.html`. Instead it would be good to put that into a new `.js` file, which is included both in the 'facebook-experiment` group and a new group for the `iebc_office_single_select.html`. (It should have "Select2" somewhere in the function name too.)